amalgamate.py

```diff
@@ -386,7 +386,7 @@ def rewrite_imports(name, pkg, order, imps):
             for n in a.names:
                 p, dot, m = n.name.rpartition('.')
                 if p == pkg and m in order:
-                    msg = ('Cannot amalgamate almagate import of '
+                    msg = ('Cannot amalgamate amalgam import of '
```

Comment:
To me, it feels like this message should just be "Cannot amalgamate import of ..." rather than "Cannot amalgamate amalgam import of".